cf/delete-sample-file-lambda.yaml

```diff
@@ -0,0 +1,110 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Setting up a lambda function to handle user invites for non-existing users
+
+Parameters:
+  Environment:
+    Type: String
+    Default: development
+    AllowedValues:
+      - development
+      - staging
+      - production
+    Description: The environment for which the tables need to be created.
+
+Outputs:
+  DeleteSampleFileLambdaArn:
+    Description: ARN of DeleteSampleFileLambdaArn
+    Value: !GetAtt DeleteSampleFileLambda.Arn
+    Export:
+      Name: !Sub DeleteSampleFileLambdaArn-${Environment}
+
+Resources:
+  DeleteSampleFileLambdaRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: "sts:AssumeRole"
+      Path: /
+      Policies:
+        - PolicyName: !Sub "can-delete-sample-files-in-${Environment}"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - 's3:DeleteObject'
+                Resource:
+                  - Fn::Sub: "arn:aws:s3:::biomage-originals-${Environment}-${AWS::AccountId}/*"
+        - PolicyName: !Sub "can-create-log-group-in-cloudwatch-${Environment}"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "logs:CreateLogGroup"
+                Resource:
+                  - Fn::Sub: "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/delete-sample-file-lambda-${Environment}"
+        - PolicyName: !Sub "can-add-logs-in-cloudwatch-${Environment}"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "logs:CreateLogStream"
+                  - "logs:PutLogEvents"
+                Resource:
+                  - Fn::Sub: "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/delete-sample-file-lambda-${Environment}:*"
+
+  DeleteSampleFileLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Sub delete-sample-labda-${Environment}
```

Comment:
Can you add expiration policy to the logs that this Lambda function creates? The logs it creates are in Cloudwatch and by default they never expire

Reply:
Added to the log group role on the last line